### PR TITLE
[75] Reset password (Frontend)

### PR DESCRIPTION
**Pull request by SaidganiUsm**
Where
[Reset password (Frontend)](https://jjbcapital.atlassian.net/browse/NL-75)

What
Forget and reset password pages

How
Designed the page for changing password for the user 

Affected Area
Auctionify.API
Auctionify.Infrastructure

PR Checklist
Build

 Build Succeeded
Unit Tests

 X Tests Passed, Y Ignored, Z Failed

### CODE REVIEW

File: src/Auctionify.API/ClientApp/src/app/api-authorization/authorize.service.ts [hunk 1]
```diff
@@ -31,7 +36,9 @@ export class AuthorizeService {
   private user: BehaviorSubject<IUser | null> =
     new BehaviorSubject<IUser | null>(null);
 
-  constructor(private client: Client) {}
+    constructor(private client: Client, private httpClient: HttpClient) {
+      //this.initializeAuthorizeService();
```
Comment (khasanrashidov) [inline src/Auctionify.API/ClientApp/src/app/api-authorization/authorize.service.ts:40]: do we need this comment?)

**Comment by khasanrashidov** [comment]
Can you merge and resolve conflicts as well? It should be guaranteed that after merge your application works))

**Comment by khasanrashidov** [comment]
Fix it:

image

image

**Review by TsvirkoNikita — CHANGES_REQUESTED** [review]
Link didn't work
image

**Review by khasanrashidov — APPROVED** [review]

**Pull request merged**